<ui version="4.0" >
 <class>Metadata</class>
 <widget class="QWidget" name="Metadata" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>218</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Metadata</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Album Art</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAlbumArt" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="scaledContents" >
      <bool>true</bool>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="styleSheet" >
        <string notr="true" >color: #555;</string>
       </property>
       <property name="text" >
        <string>Title:</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="labelTitle" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="styleSheet" >
        <string notr="true" >color: #555;</string>
       </property>
       <property name="text" >
        <string>Album:</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="labelAlbum" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="styleSheet" >
        <string notr="true" >color: #555;</string>
       </property>
       <property name="text" >
        <string>Year:</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="labelYear" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="styleSheet" >
        <string notr="true" >color: #555;</string>
       </property>
       <property name="text" >
        <string>Artist:</string>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLabel" name="labelArtist" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>384</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
